.drone.yml

```diff
@@ -0,0 +1,44 @@
+pipeline:
+  compile:
+    image: nextcloudci/android:android-33
+    commands:
+      # build app and assemble APK
+      - sh -c "if [ '${FLAVOR}' != 'Lint' ]; then ./gradlew assemble${FLAVOR}; fi"
+    when:
+      matrix:
+        FLAVOR: [Generic, Gplay]
```

Comment:
Is this because Gradle upper-cases first letter? Because the flavours are "generic" and "gplay".

Reply:
Yup, that's why - ignore me :P

Reply:
They are uppercase so I can use them directly in assemble${FLAVOR} (which then becomes assembleGplay / assemble Generic)